aci/data_source_aci_rtctrlattrp.go

```diff
@@ -21,14 +23,30 @@ func dataSourceAciActionRuleProfile() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 			},
-			"annotation": {
+			"name": {
 				Type:     schema.TypeString,
+				Required: true,
+			},
+			"set_communities": {
 				Optional: true,
-				Computed: true,
+				Type:     schema.TypeMap,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
 			},
-			"name": {
+			"next_hop_propagation": {
 				Type:     schema.TypeString,
-				Required: true,
+				Optional: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"nh-unchanged",
+				}, false),
```

Comment:
no need for validate func as this is not used for input but for adding what is configured (data source)